### DISCUSSION TITLE
How to export all client software versions as excel in one sheet?

### DISCUSSION THREAD

**Discussion by BananamanCoolDude** [amidaware/tacticalrmm#1039] [Q&A]
Hello Fellow humans!

I heard there is an option to export all agent software with their versions as excel in one sheet? 

Cannot find how to do it besides doing it manually one by one, really useful when doing audits for up-to-date software. 
Could anyone point me in the right direction? :)

Sincerely,
Banana :)

**Comment by silversword411** [comment]
What, like this?

![2022-04-01_174540 - firefox_Tactical_RMM_—_Mozilla_Firefox](https://user-images.githubusercontent.com/13395348/161345526-3316c6bb-4d43-4eaf-ba17-558034830706.png)

**Comment by BananamanCoolDude** [comment]
This is only exporting for the selected user. I need to export software list for ALL users.

**Comment by dinger1986** [comment]
Can muck about in django or db and do reports docs should give you a start on them.

Native reporting is being developed now.
